docs/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -144,7 +144,8 @@ We recommend you include your DSN directly in these three files. Alternatively y
 
 While the client initialization code will be injected into your application's client bundle by `withSentryConfig` which we set up earlier,
 the configuration for the server and edge runtime needs to be imported from a [Next.js Instrumentation file](https://nextjs.org/docs/app/building-your-application/optimizing/instrumentation).
-To set up this file, add a `instrumentation.ts` file to the root directory of your Next.js application (or inside the `src` folder if you're using one) and add the following content:
+To set up this file, enable the Next.js instrumentation hook by setting the [`experimental.instrumentationHook`](https://nextjs.org/docs/app/api-reference/next-config-js/instrumentationHook) to `true` in your `next.config.js`.
```

Comment:
Since this is required, can you also update the [code snippet for `next.config.js`](https://sentry-docs-git-sig-instrumentation-nextjs.sentry.dev/platforms/javascript/guides/nextjs/manual-setup/#extend-your-nextjs-configuration) to include this?  Otherwise I feel like it's still easy to miss for a user just skimming & copy-pasting the code samples

Reply:
It is technically not required, which is why I am also a bit on the fence about this change. Something is fishy with the user report.

Reply:
Should we add this then? I think it is a little bit more transparent if the users set this option themselves. But if the SDK enables `instrumentationHook` we could also document that this is enabled for them? Just for some context on what's going on in the background.

Reply:
I say we just merge this as is and all is fine.